### ISSUE TITLE
能不能做到外层NestedScrollView保持bouncing滚动效果

### ISSUE THREAD

**Issue by 826327700** [fluttercandies/extended_nested_scroll_view#68]
现在的情况是，无论是原生的NestedScrollView还是extended_nested_scroll_view，只要在body里含有滚动类的widget，比如listview，外层NestedScrollView都会失去bouncing滚动效果

**Comment by yasinarik** [comment]
I had to use Google translate but I can confirm you. 

Normally, I use a NestedScroller for making all of the nested ScrollViews (it can be ListView, Slivers, Grids, etc...) to scroll **as one unit**.

Thus, I would like to have a behavior that the most outer scroller uses `AlwaysScrollableScrollPhysics`. This way, it can bounce and be over-scrolled. These two behaviors are essential for UX.

Instagram uses the described behavior on its user profile screen.

I have tried many things to achieve this but I was unsuccessful.
